packages/ng/multi-select/input/select-input.component.ts

```diff
@@ -79,7 +79,7 @@ export class LuMultiSelectInputComponent<T> extends ALuSelectInputComponent<T, T
 	}
 
 	public override updateValue(value: T[], skipFocus = false): void {
-		super.updateValue(value);
+		super.updateValue(value, skipFocus);
```

Comment:
Could we rename skipFocus into something else?

Reply:
Maybe `skipInteractions`? or `skipFocusAndOpen` to make it more explicit?